graphrag/index/api.py

```diff
@@ -22,12 +22,12 @@
 
 async def build_index(
     config: GraphRagConfig,
+    emit: list[str],
```

Comment:
do you think it makes sense, at least from a conceptual standpoint, to have this param so high in the signature?

Reply:
I originally moved this due to removing the default value of `None`. It's been moved back though and the default value set to `parquet`. At the API layer, I think this makes more sense.

We don't want the defualt value to be defined as `None` at the API layer because that gets changed to `parquet` inside the library anyway. It's best to make this default behavior more transparent.